### ISSUE TITLE
Slider - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570181)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7818]
### Violation: 
 Ensure parts of graphical objects essential for understanding content have sufficient contrast 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/b4dd7833-708a-4d2b-b8a6-3f1423aa2e71)

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 

### Severity: 
 5 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: Slider - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570181) 
- Module: Slider 
**Issue**
There are graphical objects with a color contrast ratio below 3.00:1.

Foreground: #EAEAEA
Background: #F8F8F8
Contrast ratio: 1.13:1

Examples include:
- The gray part of all the images, in "Test 8. Slider with Histogram" section.

**User Impact**
Users with low vision will have difficulty identifying this content.

 
**Suggestion**
Ensure parts of graphical objects essential for understanding content have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying objects include lines in a chart, meaningful icons, and annotations within an image.

Graphics that require particular presentation to preserve their meaning are exempt from this requirement.

To evaluate color contrast, see the Accessible Color Picker extension: https://www.accessibility.dev/

 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Closing in favor of https://github.com/Esri/calcite-design-system/issues/7822 for a more comprehensive audit across the design system.
